example/reinforcement-learning/dqn/dqn_demo.py

```diff
@@ -23,7 +23,7 @@
 from base import Base
 from operators import *
 from atari_game import AtariGame
-from utils import *
+import utils
```

Comment:
If you can pick up the two changes in #11994 then I can close #11994.  It is funny the way that flake8 and pylint detect different undefined names.

Reply:
pylint did detect the errors from sys and time. but dint make the changes here because it was already covered in your PR. I'll merge those changes to this PR and resubmit.